OSX/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>1</string>
```

Comment:
Should this (and other occurrences) follow the library versioning, i.e. 6.0 for this)?  And if so should we update the release process to automatically apply that?